### PR TITLE
[Customer] - add a token-secured Customer resource

### PR DESCRIPTION
**Pull request by NoBrainSkull**

### CODE REVIEW

**Comment by cyrilchapon** [comment]
Hi,

Your help is greatly appreciated =)

Great work digging into sails.js!

Please, review the following little things so I can merge that good work ASAP :
- [`ResourceController`](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/65723f0493c4704447ece92899df62cfbd9afdde#diff-96e7ad6cf4920dd0faeea6d1bc80fa63R2) should be called `CustomerController` (please mind also renaming comments)
- [Here](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/65723f0493c4704447ece92899df62cfbd9afdde#diff-96e7ad6cf4920dd0faeea6d1bc80fa63R9) (and [here](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/7dbd73deba8f0c7023693bbd2d351d8d0922efa4#diff-6373145b9109eb761a7c3123b49b53ceR9)), we don't need a [controller action](http://sailsjs.org/documentation/concepts/controllers#?actions) to obtain the RESTful `GET` on `/customer` ressource : sails [automatically exposes RESTful routes](http://sailsjs.org/documentation/reference/blueprint-api#?restful-routes) for any well-defined _Model_ (check _User_ Controller, empty but still accessible on `GET /user`)
- Remove ghost modifications from commits, like [this one](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/65723f0493c4704447ece92899df62cfbd9afdde#diff-99dda4d331e26142c061301ab9a9dd6bR19) and [this one](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/65723f0493c4704447ece92899df62cfbd9afdde#diff-aaaa74610a813d59bae34f335326bd25R20), [this one](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/65723f0493c4704447ece92899df62cfbd9afdde#diff-aaaa74610a813d59bae34f335326bd25R30) and [this one](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/65723f0493c4704447ece92899df62cfbd9afdde#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R31)
- Regarding [Customer Model](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/65723f0493c4704447ece92899df62cfbd9afdde#diff-20181138673a4f8ba1c4c58cfaf25779R1), please use [standard syntax](http://sailsjs.org/documentation/concepts/models-and-orm/models). We don't need anything else, table [identity](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/65723f0493c4704447ece92899df62cfbd9afdde#diff-20181138673a4f8ba1c4c58cfaf25779R3), [autoPk](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/65723f0493c4704447ece92899df62cfbd9afdde#diff-20181138673a4f8ba1c4c58cfaf25779R9) and [storage (connection)](https://github.com/cyrilchapon/exia-secu-demo-api/pull/2/commits/65723f0493c4704447ece92899df62cfbd9afdde#diff-20181138673a4f8ba1c4c58cfaf25779R4) are automatically handled by sails
- [optional] Split commits into small, semantic and clean commits

Basically, for models, just define attributes unless you want special stuff. And always try to keep sails default behavior on implicit REST controllers !
The CORS logic is ok.

**Comment by NoBrainSkull** [comment]
Hi,

Thanks for your feedback. I made the changes you asked for.

Cheers

**Comment by cyrilchapon** [comment]
Awesome.
